.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "wavm"]
```

Comment:
is it final?

Reply:
Nope, will merge PR adding WAVM into soramitsu hunter repo today and then switch wavm in kagome to hunterized version instead of a submodule.